src/Symfony/Component/Validator/ExecutionContextInterface.php

```diff
@@ -1,319 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Symfony\Component\Validator;
-
-/**
- * Stores the validator's state during validation.
```

Comment:
not sure what part of the documention is still relevant for the new documentation, which doesn't exist there